packages/aws-cdk-lib/aws-synthetics/lib/canary.ts

```diff
@@ -269,7 +269,7 @@ export interface CanaryProps {
    * Artifact encryption is only supported for canaries that use Synthetics runtime
    * version `syn-nodejs-puppeteer-3.3` or later.
    *
-   * @default - Artifacts are encrypted at rest using an AWS managed key
+   * @default - `ArtifactsEncryptionMode.KMS` is set if you specify `artifactS3KmsKey`, otherwise artifacts are encrypted at rest using an AWS managed key
```

Comment:
nit: I think stating the normal default "Artifacts encrypted at rest with AWS-managed key" first might be better phrasing, mentioning the `artifactS3KmsKey` afterwards, just to avoid confusion.